draft-ietf-taps-interface.md

```diff
@@ -1904,47 +1904,70 @@ options.
 
 Applicability:
 : Preconnection, Connection, Message
 
-This property specifies the application's expectation of the dominating traffic pattern for this Connection.
-This implies that the transport system should optimize for the capacity profile specified. This can influence path and protocol selection.
-The following values are valid for the Capacity Profile:
+This property specifies the traffic interaction pattern expected by the
+application, the desired network treatment for traffic sent by the application,
+and the tradeoffs the application is prepared to make in path and protocol
+selection to receive that desired treatment. When the capacity profile is set to
+a value other than Default, the transport system should select paths and
+profiles to optimize for the capacity profile specified. The following values
+are valid for the Capacity Profile:
 
   Default:
-  : The application makes no request about its expected
-  capacity profile. No special optimizations of the tradeoff between
-  delay, delay variation, and bandwidth efficiency should be made when selecting and
-  configuring stacks.
-
-  Low Latency:
-  : Response time (latency) should be optimized at
-  the expense of bandwidth efficiency and delay variation when sending this
-  message. This can be used by the system to disable the coalescing of
-  multiple small Messages into larger packets (Nagle's algorithm); to prefer
-  immediate acknowledgment from the peer endpoint when supported by the
-  underlying transport; to signal a preference for lower-latency, higher-loss
-  treatment; and so on.
-
-  Constant Rate:
-  : The application expects to send/receive data at a constant rate after
-  Connection establishment. Delay and delay variation should be minimized at the
-  expense of bandwidth efficiency. This implies that the Connection may fail
-  if the desired rate cannot be maintained across the Path. A transport
-  may interpret this capacity profile as preferring a circuit breaker
-  {{?RFC8084}} to a rate-adaptive congestion controller.
-
-  Scavenger/Bulk:
-  : The application is not interactive. It expects to send/receive a large
-  amount of data, without any urgency. This can, for example, be used to select protocol
-  stacks with scavenger transmission control, to signal a preference for
-  less-than-best-effort treatment, or to assign the traffic to a lower-effort service.
-
+  : The application makes no representation about its expected capacity
+  profile. No special optimizations of the tradeoff between delay, delay
+  variation, and bandwidth efficiency should be made when selecting and
+  configuring transport protocol stacks. When the underlying transport protocol
+  stack(s) support per-connection DSCP signaling, transmitted traffic SHOULD carry
+  the DSCP Best Effort PHB. \[EDITOR'S NOTE: cite]
+
+  Scavenger/Bulk: 
+  : The application is not interactive. It expects to send
+  and/or receive a large amount of data, without any urgency. This can, for
```

Comment:
This was also in the old text and I only notice it now: why do we say "a large amount of data"?  It could also be tiny?  That seems quite unnecessary here.

Reply:
agreed, will fix.

Reply:
Actually, I find Scavenger/Bulk very confusing. Bulk is a large amount of data, and could still be urgent. Scavenger could be tiny or large and is not urgent. So if this is what we are talking about the name should be only Scavenger?